### PR TITLE
iptables, nft: add masquerade example

### PR DESCRIPTION
**Pull request by Lodifice**
If find the masquerade (NAT) functionality to be the one I actually use most, and it wasn't documented here.
I have to look it up myself sometimes and I think that other user can also benefit from the example.
For the nft page, it's a pity that you have to issue multiple commands (since a `nat` table or `postrouting` chain do not exist by default), but the TLDR guidelines don't allow multiple commands per example.

- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/linux/iptables.md [hunk 1]
```diff
@@ -19,6 +19,10 @@
 
 `sudo iptables -A {{chain}} -s {{ip}} -p {{protocol}} --dport {{port}} -j {{rule}}`
 
+- Add a rule to NAT all traffic from the 192.168.0.0/24 subnet to the host's public IP:
```
Comment (marchersimon) [inline pages/linux/iptables.md:22]:
```suggestion
- Add a NAT rule to translate all traffic from the `192.168.0.0/24` subnet to the host's public IP:
```
35531403820b148772e76c2e94e4856aff5ac258 should also be applied here and using Backticks for the IP will make it a different color and make it more readable.

File: pages/linux/nft.md [hunk 2]
```diff
@@ -19,6 +19,10 @@
 
 `sudo nft add rule {{inet}} {{filter}} {{input}} {{tcp}} {{dport \{ telnet, ssh, http, https \} accept}}`
 
+- Add a NAT rule to translate all traffic from the 192.168.0.0/24 subnet to the host's public IP:
```
Comment (marchersimon) [inline pages/linux/nft.md:22]:
```suggestion
- Add a NAT rule to translate all traffic from the `192.168.0.0/24` subnet to the host's public IP:
```

**Comment by Lodifice** [comment]
Darn, I overlooked the wording for iptables. Thanks a lot for your suggestions!

**Review by marchersimon — APPROVED** [review]
Thanks @Lodifice

**Review by sbrl — APPROVED** [review]
Great addition. Thanks, @Lodifice!

**Pull request merged**